functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SurefireForksConfigurationTest.java

```diff
@@ -71,9 +75,13 @@ private void verifyTestSuiteExecution(String... systemPropertiesPairs){
         // when
         final List<TestResult> actualTestResults =
             project
-                .build("config/impl-base")
+                .build()
                 .options()
+                    //.withRemoteDebugging()
+                    //.withRemoteSurefireDebugging()
                     .withSystemProperties(systemPropertiesPairs)
+                    .withSystemProperties("graph.name", name.getMethodName())
+                    .withSystemProperties("smart.testing.debug", "true") // This will only be propagated to surefire for "not_reusing_forks" option
```

Comment:
wouldn't be good to have these properties stored as constants somewhere...?

Reply:
Sure, it's still WIP as it needs to be aligned with Debug Mode once we have it.

Reply:
then add the label :-P

Reply:
or read the description :P 

> Graph is dumped when debug mode is enabled (should be aligned with #91)